endpointmanager/pkg/endpointmanager/fhirendpoint.go

```diff
@@ -57,7 +58,18 @@ func (e *FHIREndpoint) Equal(e2 *FHIREndpoint) bool {
 	if e.TLSVersion != e2.TLSVersion {
 		return false
 	}
-	if e.MimeType != e2.MimeType {
+
+	// check MIMETypes equal
+	if len(e.MIMETypes) != len(e2.MIMETypes) {
+		return false
+	}
+	for i, v := range e.MIMETypes {
+		if v != e2.MIMETypes[i] {
```

Comment:
So the mime types have to be in the same order in both endpoints in order for them to be considered equal?